src/Compiler/Service/IncrementalBuild.fs

```diff
@@ -285,7 +303,6 @@ type BoundModel private (tcConfig: TcConfig,
                          keepAssemblyContents, keepAllBackgroundResolutions,
                          keepAllBackgroundSymbolUses,
                          enableBackgroundItemKeyStoreAndSemanticClassification,
-                         enablePartialTypeChecking,
```

Comment:
I'm a bit lost by this removal.
Are you now always assuming partial type checking?